include/xten/Transform/XTenCanonicalizePass.h

```diff
@@ -11,7 +11,7 @@ namespace xilinx::xten {
 #include "xten/Transform/Passes.h.inc"
 
 /// Populate \p patterns with XTen concat folding patterns.
-void populateXTenFoldConcatPatterns(mlir::RewritePatternSet &patterns, mlir::PatternBenefit benefit = 1);
+void populateXTenQDQConcatPatterns(mlir::RewritePatternSet &patterns);
```

Comment:
Can we add a big warning here and in the source code of the pattern that this is destructive and changes the semantics of the IR?